src/net.cpp

```diff
@@ -1971,7 +1971,8 @@ bool OpenNetworkConnection(
         LOCK(cs_vNodes);
         // Dandelion: new outbound connection
         CNode::vDandelionOutbound.push_back(pnode);
-        if (CNode::vDandelionDestination.size() < DANDELION_MAX_DESTINATIONS) {
+        auto consensus = Params().GetConsensus();
```

Comment:
Here a non-const copy of Consensus are created and the `consensus` object can be changed later. `const auto &` works ways better here as it will store the reference that `const Consensus::Params& GetConsensus() const` returns.

Reply:
Agree!